src/index

```diff
@@ -3,20 +3,20 @@
  <head>
   <title>Web Hypertext Application Technology Working Group</title>
   <link rel="stylesheet" href="/style/tabbed-pages">
-  <link rel="icon" href="/images/icon">
+  <link rel="icon" href="https://resources.whatwg.org/logo.svg">
  </head>
  <body class="front-page">
   <hgroup>
-   <h1><object data="https://images.whatwg.org/logo.svg" height=101 width=101><img src="https://images.whatwg.org/logo" alt="" height=101 width=101></object> Welcome to the WHATWG community</h1>
+   <h1><img src="https://resources.whatwg.org/logo.svg" height=101 width=101 alt> Welcome to the WHATWG community</h1>
    <h2>Maintaining and evolving HTML since 2004</h2>
   </hgroup>
 
   <a href="https://wiki.whatwg.org/wiki/FAQ"><p><strong>FAQ</strong> <span>Get answers</span> <span>to your questions</span> <span>about HTML</span> <span>and the WHATWG</span></p></a>
   <a href="https://html.spec.whatwg.org/multipage/"><p><strong>HTML</strong> <span>Read, use,</span> <span>or implement</span> <span>the HTML Living Standard</span></p></a>
-  <a href="https://whatwg.org/specs/"><p><strong>Specs</strong> <span>See the other</span> specifications <span>developed at the WHATWG</span></p></a>
+  <a href="https://whatwg.org/specs/"><p><strong>Standards</strong> <span>See the other</span> standards <span>developed at the WHATWG</span></p></a>
 
-  <a href="https://developers.whatwg.org/"><p><strong>Web Dev Edition</strong> <span>Read an edition of</span> <span>the HTML standard</span> <span>designed for</span> <span>Web Developers</span></p></a>
-  <a href="http://html5.validator.nu/"><p><strong>Validator</strong> <span>Check your documents</span> <span>with an HTML validator</span></p></a>
+  <a href="https://blog.whatwg.org/"><p><strong>News</strong> <span>Read and contribute</span> <span>to the WHATWG blog</span></p></a>
+  <a href="https://checker.html5.org/"><p><strong>Html Checker</strong> <span>Validate your documents</span> <span>with an HTML checker</span></p></a>
```

Comment:
> Why is it that way in the checker service, I wonder?

Because it looks better in the full context: _Nu Html Checker_ and I like it better that way. But again I don’t feel strongly about forcing anybody else to replicate that faithfully in references.